bindings/python/tests/test_services.py

```diff
@@ -125,11 +125,12 @@ async def test_async_write(self):
         filename = f"test_file_{str(uuid4())}.txt"
         content = os.urandom(size)
         size = len(content)
-        await self.async_operator.write(filename, content)
+        await self.async_operator.write(filename, content, content_type='text/plain')
```

Comment:
Please don't touch existing test cases, add new one instead. And not all service support `write_with_content_type`, we need to check the capability first. Maybe we can add this in next PR.